### ISSUE TITLE
[vcpkg_acquire_msys.cmake] missing download of gettext

### ISSUE THREAD

**Issue by StarGate-One** [microsoft/vcpkg#13291]
**Describe the bug**
[vcpkg_acquire_msys.cmake] missing download of gettext

**Environment**
- OS: [Microsoft Windows [Version 10.0.19041.450]]
- Compiler: Visual Studio  Version: 16.7.2  VS Tools Version: 14.27.29110

**To Reproduce**
1. `.\vcpkg.exe install libpq[core,icu,nls,openssl,xml,zlib]:x64-windows`
2. `Unknown packages were required for vcpkg_acquire_msys(gettext)`

**Expected behavior**
It should download the appropriate tool gettext from msys

**Failure logs** 
None

**Additional context**
PR forthcoming

**Comment by StarGate-One** [comment]
I close ticket since I not know how to fix.

**Comment by NancyLi1013** [comment]
Hi @StarGate-One 

This issue still needs to be fixed, isn't it? If so, we should keep it open until it is fixed.

**Comment by StarGate-One** [comment]
@NancyLi1013 
@BillyONeal merged the fix after @ras0219 approval 3 days ago.

**Comment by StarGate-One** [comment]
PS I did not close. The PR merge did.

**Comment by StarGate-One** [comment]
Yes there should be a PR and ticket still open for libpq as it will fail if decimal-for-cpp is already installed.
The other issue with libpq was with imgui which also was fixed by @strega-nil a few days ago.

**Comment by NancyLi1013** [comment]
Yes, I know this is closed automatically. I want to confirm with you that either you want to keep this issue open or open a new issue ticket to track `libpq `issue.

Thanks.

**Comment by StarGate-One** [comment]
The libpq issues I had have been fixed. This one just happened to come to light when I installed libpq, but this issue would arise on any port using msys2 needing the gettext package. The issue was not libpq but the cmake script downloading the msys2 packages in download mode after the conversion was made to not use the full msys2 package, but download only those needed.
So it is ok for this to be closed.
